python/hail/api2/matrixtable.py

```diff
@@ -734,15 +731,15 @@ def select_cols(self, *exprs, **named_exprs):
 
         for e in exprs:
             all_exprs.append(e)
-            analyze(e, self._col_indices, {self._row_axis})
+            analyze('MatrixTable.select_cols', e, self._col_indices, {self._row_axis})
```

Comment:
you might as well move `def select_rows` above `def select_cols` in this PR so it parallels all the others